### PR TITLE
Update bj-share categories

### PR DESCRIPTION
**Pull request by dctorrent1337**

### CODE REVIEW

**Review by garfield69 — CHANGES_REQUESTED** [review]
Categories 19 and 20 are already in use earlier in the category table.
Can you please examine all the category ids and cross check. 
There cannot be duplicate ids for different category types.
Thanks.

**Comment by dctorrent1337** [comment]
It there any reason I cannot use TorznabCatType.BooksEbook and TorznabCatType.BooksMagazines? I have seen them in other trackers definitions. 

```
...
/home/vsts/work/1/s/src/Jackett.Common/Indexers/BJShare.cs(106,50): error CS0117: 'TorznabCatType' does not contain a definition for 'BooksMagazines' [/home/vsts/work/1/s/src/Jackett.Common/Jackett.Common.csproj]
/home/vsts/work/1/s/src/Jackett.Common/Indexers/BJShare.cs(107,51): error CS0117: 'TorznabCatType' does not contain a definition for 'BooksEbook' [/home/vsts/work/1/s/src/Jackett.Common/Jackett.Common.csproj]
...
```

**Comment by garfield69** [comment]
Have a look at https://github.com/Jackett/Jackett/wiki/Jackett-Categories for the valid category names, and in particular, the last section about C# names ;-)

File: src/Jackett.Common/Indexers/BJShare.cs [hunk 1]
```diff
@@ -95,28 +95,29 @@ public class BJShare : BaseWebIndexer
...
             AddCategoryMapping(4, TorznabCatType.PCGames, "Jogos");
-            AddCategoryMapping(199, TorznabCatType.XXX, "Filmes Adultos");
-            AddCategoryMapping(200, TorznabCatType.XXX, "Jogos Adultos");
-            AddCategoryMapping(201, TorznabCatType.XXXImageSet, "Fotos Adultas");
+            AddCategoryMapping(5, TorznabCatType.BooksComics, "Mangás");
+            AddCategoryMapping(6, TorznabCatType.TV, "Vídeos de TV");
+            AddCategoryMapping(7, TorznabCatType.Other, "Outros");
+            AddCategoryMapping(8, TorznabCatType.TVSport, "Esportes");
+            AddCategoryMapping(9, TorznabCatType.BooksMags, "Revistas");
+            AddCategoryMapping(10, TorznabCatType.Books, "E-Books");
```
Comment (garfield69) [inline src/Jackett.Common/Indexers/BJShare.cs:107]:
```suggestion
            AddCategoryMapping(10, TorznabCatType.BooksEBook, "E-Books");
```

**Review by garfield69 — CHANGES_REQUESTED** [review]

**Review by garfield69 — APPROVED** [review]

**Comment by garfield69** [comment]
Thank you.

**Comment by DaniloNC** [comment]
Thank YOU!

**Comment by garfield69** [comment]
v0.20.294

**Pull request merged**